src/Uno.UI/UI/Xaml/Controls/MediaPlayerElement/HtmlMediaPlayer.wasm.cs

```diff
@@ -0,0 +1,413 @@
+using System;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using Uno.Extensions;
+using Uno.Foundation;
+using Uno.Foundation.Logging;
+using Windows.Media.Playback;
+using NativeMethods = __Windows.Devices.Media.MediaElement.NativeMethods;
+
+namespace Windows.UI.Xaml.Controls;
+
+internal class HtmlMediaPlayer : Border, IHtmlMediaPlayer
```

Comment:
This is fine if this overall new feature is only available for net7 or higher, because of the dependency on JSImport.